### PR TITLE
BORS: welcome to aya

### PR DESCRIPTION
**Pull request by imkiva**

### CODE REVIEW

**Comment by imkiva** [comment]
bors try

**Comment by ice1000** [comment]
@imkiva The CI is not enabled for bors I guess
image
See? No orange circle

File: .github/workflows/vsix.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
-name: build vsix
+name: vsix
 
 on:
   push:
-    branches: [ main ]
+    branches: [main, trying, staging]
```
Comment (ice1000) [inline .github/workflows/vsix.yml:5]: Yes
YES
Kiva got the right configurations

**Comment by ice1000** [comment]
bors try

**Comment by ice1000** [comment]
I guess you wanna try to change the `vsix` to `build`.

**Comment by ice1000** [comment]
> Build succeeded:
> 
> * [build (15.x)](https://github.com/aya-prover/aya-vscode/runs/3934115052)

🙏️✌️👌️🙋️
Thanks!

File: bors.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+status = ["build (15.x)"]
```
Comment (imkiva) [inline bors.toml:1]: LMAO

Comment (ice1000) [reply bors.toml:1]: 🤦‍♀️

**Pull request merged**